DuckDuckGo/TabViewController.swift

```diff
@@ -48,6 +48,15 @@ class TabViewController: WebViewController {
         webEventsDelegate = self
     }
     
+    public var link: Link? {
+        guard let url = url else {
+            return Link.mergeData(primary: nil, secondary: tabModel.link)
```

Comment:
Given this function just returns the secondary if the primary is nil anyway, could just return the `tabModel.link` in this case and simplify `mergeData`?